content/en/docs/ops/configuration/traffic-management/dns/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Understanding DNS
+linktitle: DNS
+description: How DNS interacts with Istio.
+weight: 30
+keywords: [traffic-management,proxy]
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+Istio interacts with DNS in various different ways that can be confusing to understand.
+This document provides a deep dive into how Istio and DNS work together.
+
+{{< warning >}}
+This document describes low level implementation details. For a higher level overview, check out the traffic management [Concepts](/docs/concepts/traffic-management/) or [Tasks](/docs/tasks/traffic-management/).
+{{< /warning >}}
+
+## Life of a request
+
+In these examples, we will walk through what happens when an application runs `curl example.com`.
+While `curl` is used here, the same applies to almost all clients.
+
+When you send a request to a domain, a client will do DNS resolution to resolve that to an IP address.
+This happens regardless of any Istio settings, as Istio only intercepts networking traffic; it cannot change your application's behavior or decision to send a DNS request.
+In the example below, `example.com` resolved to `192.0.2.0`.
```

Comment:
I think it would be good to explain a little more here about how this happens, e.g., using `/etc/resolve` DNS configuration, kube-dns inside a kube cluster. Also would also be good to mention that it might not be able to resolve a host, which is where service entries come into play?

Reply:
That is the point I am trying to make - that Istio does NOT impact this at all. ServiceEntries do NOT come into play for the 99% of users not using the DNS proxy which is discussed more below. Will try to make this more clear

Reply:
Yeah, but it seems like the rest of the document is always talking about the original IP address the client sent, without mentioning the case where the client DNS couldn't resolve, which is why lots of what is described later on is important. Maybe just clarify that this example assumes that the platform DNS successfully resolved the host but if not, then on line 33 `At this point, Istio will see both ...` would not be true, it would only have the host, so the service entry can not use `resolution: NONE`

Reply:
If they client can't resolve then thats the end of the story. There is no network request, so Istio doesn't know/care. Will add that to the doc

Reply:
Perfect! Tying in (mentioning) the `## DNS Proxying` section below would also help, since otherwise the document flow is from client DNS, to Istio networking, and then back to client DNS ... which is a little confusing IMO.